controllers/feeds/getAnonymousFeedChat.js

```diff
@@ -16,8 +16,7 @@ const getAnonymousFeedChatService = async (req, res) => {
   try {
     let {last_fetch_date = null} = req.query;
     if (last_fetch_date) {
-      last_fetch_date = new Date(last_fetch_date);
-      last_fetch_date = last_fetch_date.toISOString();
+      last_fetch_date = moment.utc(last_fetch_date).format('YYYY-MM-DDTHH:mm:ss');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The conversion of `last_fetch_date` to UTC format using `moment.js` is a good approach. However, it's important to handle the scenario where the date provided might not be valid. If an invalid date is passed, `moment.utc(last_fetch_date)` will return an invalid date object which could lead to unexpected behavior in subsequent code. Consider adding error handling for this case.
```diff
+    if (last_fetch_date) {
+      const utcDate = moment.utc(last_fetch_date);
+      if (!utcDate.isValid()) {
+        return res.status(400).send({
+          success: false,
+          message: 'Invalid last_fetch_date'
+        });
+      }
+      last_fetch_date = utcDate.format('YYYY-MM-DDTHH:mm:ss');
+    }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->